packages/api-extractor/src/generators/ExcerptBuilder.ts

```diff
@@ -162,6 +183,8 @@ export class ExcerptBuilder {
 
 			if (canonicalReference) {
 				ExcerptBuilder._appendToken(excerptTokens, ExcerptTokenKind.Reference, span.prefix, canonicalReference);
+			} else if (ExcerptBuilder.isPrimitiveKeyword(span.node)) {
+				ExcerptBuilder._appendToken(excerptTokens, ExcerptTokenKind.Reference, span.prefix);
```

Comment:
Currently the website assumes all Reference ExcerptTokens have a canonicalReference. So this would either need to now be checked for in ExcerptText.tsx or we need canonicalReferences for primite keywords

Reply:
I would lean towards doing it on the website. We can assume that a reference node with a nullish canonicle reference is considered a primitive.

Reply:
I wouldn't "assume" it, but try to look it up in the link table of #9913 and if it's not in there handle it like Content.

Reply:
This is what I meant. Instead of performing a check on each content node we can narrow it down only to nodes that are both references and have no canonical reference *and only* then check it against the table of keywords to see if it will be linked or not.